MAUI/Calendar/migration.md

```diff
@@ -308,12 +308,12 @@ this.Content = calendar;
 </tr>
 <tr>
 <td>{{'[FirstDayofWeek](https://help.syncfusion.com/cr/xamarin/Syncfusion.SfCalendar.XForms.SfCalendar.html#Syncfusion_SfCalendar_XForms_SfCalendar_FirstDayofWeek)'| markdownify }}</td>
-<td>{{'[FirstDayOfWeek(From MonthView)](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.Calendar.CalendarMonthView.html#Syncfusion_Maui_Calendar_CalendarMonthView_FirstDayOfWeek)'| markdownify }}</td>
+<td>{{'[[FirstDayOfWeek](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.Calendar.CalendarMonthView.html#Syncfusion_Maui_Calendar_CalendarMonthView_FirstDayOfWeek)(From [MonthView](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.Calendar.CalendarMonthView.html))]'| markdownify }}</td>
```

Comment:
Check the links are correct?

**https://help.syncfusion.com/cr/maui/Syncfusion.Maui.Calendar.CalendarMonthView.html** - specified on twice